app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="full-height bg-img">
+  <div class="row">
+    <div class="small-12 medium-10 medium-centered large-8 large-centered columns">
+
+      <div class="floating-container">
+        <h1>Users</h1>
+      </div>
+      <div class="floating-container">
+        <% @users.each do |user| %>
+          </h4><%= link_to user.username, user_path(user) %></h4>
+          <%= button_to "Delete User", "/admin/users/#{user.id}", class: "button", method: "delete", id: user.id %>
+        <% end %>
+      </div>
```

Comment:
You nested this resource in your routes, so don't hesitate to use the named route: `<%= button_to "Delete User", admin_user_path(user), ... %>`